### ISSUE TITLE
Autocomplete for emoji

### ISSUE THREAD

**Issue by gabrielqmatos88** [HabitRPG/habitica#4040]
would be nice if there was autocomplete for emoji
##

**Comment by negue** [comment]
Yes, already tried a first version of this feature but it wasn't good enough (for me) and had some bugs: #3589
I'll try a second approach with ment.io, see #2359

**Comment by crookedneighbor** [comment]
I would love this.

**Comment by librarianmage** [comment]
What is the status on this?

**Comment by Alys** [comment]
I think we should leave this one until we've finished the website improvement project. It will be involve changes to client-side code and if it were done now it would need to be rewritten for the new website code. I'm marking this as on hold.

**Comment by lemoness** [comment]
We are now open to this feature again :)

**Comment by PetalsOnWind** [comment]
Can I give this a try? (I am quite new to this and don't want to mess with something major)

**Comment by Alys** [comment]
@PetalsOnWind Sure! Thank you! Comment here if you have questions.

**Comment by beffymaroo** [comment]
@PetalsOnWind Are you still working on this issue? Please let us know. :)

**Comment by PetalsOnWind** [comment]
I had some exams lately and didn't really work on this

**Comment by beffymaroo** [comment]
@PetalsOnWind Thank you for responding! I've added the "Help Wanted" label to this issue for now, but if you'd like to try again let us know.

**Comment by veeeeeee** [comment]
Removing help wanted. Open to this feature but deprioritizing in favor of more impactful changes.

**Comment by dylphuts** [comment]
If this issue is open I would love to look into it.

**Comment by Tressley** [comment]
This was fixed in release v5.46.1. https://github.com/HabitRPG/habitica/releases/tag/v5.46.1
